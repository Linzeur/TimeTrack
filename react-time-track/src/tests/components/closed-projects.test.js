import React from "react";
import { render, wait, asFragment } from "@testing-library/react";

import ClosedProjects from "../../components/closed-projects";

const closedProjects = [
  {
    name: "Project Rumi",
    category: "Eduaction",
    client: "MINEDU",
    closed: true,
    id: 1
  }
];

test("Testing API history", async () => {
  fetch.mockResponseOnce(JSON.stringify(closedProjects));

  const { asFragment, getAllByRole } = render(<ClosedProjects />);
  // let cards;
  // await wait(() => {
  //   cards = getAllByRole("listitem");
  // });

  // expect(cards.length).toEqual(1);
  expect(asFragment()).toMatchSnapshot();
});
