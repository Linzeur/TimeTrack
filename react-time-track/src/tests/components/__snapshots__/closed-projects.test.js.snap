// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing API history 1`] = `
<DocumentFragment>
  <main>
    <section
      class="css-10oyw64-Section"
      role="list"
    />
  </main>
</DocumentFragment>
`;
